### PR TITLE
Fix non-determinism, version 3.3.1

### PR DESCRIPTION
**Pull request by Protryon**
Fixes non-determinism arising from order-dependent `HashTable` iteration, version 3.3.1.

### CODE REVIEW

File: src/test/java/com/shapesecurity/bandolier/es2017/Test262.java [hunk 1]
```diff
@@ -404,7 +409,13 @@ public void testTest262() throws Exception {
 		Path root = Paths.get(testsDir);
 		Files.walk(root).forEach(path -> {
 			try {
-				runTest(root, path);
+				int hash = runTest(root, path);
+				for (int i = 0; i < 2; ++i) {
+					int localHash = runTest(root, path);
```
Comment (bakkot) [inline src/test/java/com/shapesecurity/bandolier/es2017/Test262.java]: I would prefer we not actually evaluate the test on subsequent runs. Ideally we'd split out the "bundle" portion from the "execute" portion, and only perform the "bundle" portion here.

Comment (Protryon) [reply src/test/java/com/shapesecurity/bandolier/es2017/Test262.java]: Updated

**Comment by bakkot** [comment]
This seems like a patch, not minor?

**Review by bakkot — APPROVED** [review]
LGTM, but should probably be 3.3.1 rather than 3.4.0

**Pull request merged**